### PR TITLE
updates for better acceptance test support

### PR DESCRIPTION
**Pull request by trescube**
added support for normalizers in acceptance tests at suite and test level
added base set of normalizers

### CODE REVIEW

File: lib/eval_test.js [hunk 1]
```diff
@@ -40,9 +43,40 @@ function makeTestContext( testCase, context ) {
...
+    }
+
+  }
+
+  context.propertyNormalizers = propertyNormalizers;
+
   return context;
 }
 
+function composeNormalizer(normalizers) {
```
Comment (dianashk) [inline lib/eval_test.js:84]: would love a comment here to explain how this compose thing works

Comment (trescube) [inline lib/eval_test.js:84]: I included a comment above where it's called but I can move it.

File: lib/eval_test.js [hunk 2]
```diff
@@ -40,9 +43,51 @@ function makeTestContext( testCase, context ) {
...
+//  composeNormalizer creates a function that is the same as a(b(c(value)))
+//
+// The list must be reversed because compose iterates forward thru the array
+//  whereas we want the original order to be maintained.  That is, if [a, b, c]
+//  is specified,
+//  we want: c(b(a(value))) to maintain the order specified
+//  and not: a(b(c(value)))
+//
+// for reduce, the seed value is a function that just returns the input
+function composeNormalizer(normalizers) {
```
Comment (orangejulius) [inline lib/eval_test.js:84]: this is pretty awesome!

File: test/eval_test.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ var evalTest = require( '../lib/eval_test' );
 
 tape( 'evalTest() evaluates all edge cases correctly', function ( test ){
   var tests = [
-    {
+/*    {
```
Comment (orangejulius) [inline test/eval_test.js]: Hey! uncomment my test :P

**Comment by dianashk** [comment]
:+1:

**Comment by trescube** [comment]
Unless someone has a problem with this, I'll merge in 10 minutes.

**Comment by orangejulius** [comment]
merge it!

**Comment by trescube** [comment]
merged
